### PR TITLE
add additional column

### PR DESCRIPTION
**Pull request by Palllke2015**
Additional columns for Members list #2888

### CODE REVIEW

File: packages/ui/src/memberships/components/MemberReferrer/MemberReferre.tsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
+import React from 'react'
```
Comment (WRadoslaw) [inline packages/ui/src/memberships/components/MemberReferrer/MemberReferre.tsx:1]: Typo in file name

**Review by WRadoslaw — CHANGES_REQUESTED** [review]
Hey! Nice work and thanks for PR! From my side: 

1. Since throughout project we keep styled components under the function components let's keep doing it for the sake of uniformity. If the components you created are used in more than one place and you really think that they deserve seperate file, as I remeber this kind of files are just named `components` instead of `Fields`.

2. I think istead of creating new fragment, query, cast and interface just to fetch one more field is kinda overkill. I suspect that you approach was based on not overfetching that graphql offers, but on the other adding so much code to use it one place when it differ by only one parameter isn't desired. IMHO we should just add `invitedBy` to the `MemberWithDetailsFields` and remove `MemberAllWithDetailsFields` and its corresponding interface `MemberWithReferrer`. The overfetch won't be so big, since not many users use invite function and if they do numbers are pretty small.

**Review by WRadoslaw — CHANGES_REQUESTED** [review]
Unfortunately it's still not it. If I mislead you somehow, sorry. The main goal is to not touch almost any casting function. The only change in graphql file was to add
```
invitedBy {
    ...MembersField
}
```
to the `MemberWithDetailsFields`. I see that i forgot to mention that in fact you will need to add new query to fetch all users (same as `GetMembers`, but for `MemberWithDetailsFields` fragment. Then it's only the matter of creating new hook for it and adding invitedBy to the interface and cast (in cast u can use `invitedBy: fields.invitedBy.map(asMember)` if I'm correct) and it the end just use `MemberWithDetails` interface throught your solution.

**Comment by Palllke2015** [comment]
> Unfortunately it's still not it. If I mislead you somehow, sorry. The main goal is to not touch almost any casting function. The only change in graphql file was to add
> 
> ```
> invitedBy {
>     ...MembersField
> }
> ```
> 
> to the `MemberWithDetailsFields`. I see that i forgot to mention that in fact you will need to add new query to fetch all users (same as `GetMembers`, but for `MemberWithDetailsFields` fragment. Then it's only the matter of creating new hook for it and adding invitedBy to the interface and cast (in cast u can use `invitedBy: fields.invitedBy.map(asMember)` if I'm correct) and it the end just use `MemberWithDetails` interface throught your solution.

It's ok. I will update my PR.

**Pull request closed without merging**